games/TicTacToe/project/tictactoe-contract/src/interface.sw

```diff
@@ -0,0 +1,33 @@
+library interface;
+
+abi Game {
+    /// Starts a new game.
+    ///
+    /// # Arguments
+    ///
+    /// * `player_one`: the first player to make a move.
+    /// * `player_two`: the second player to make a move.
+    ///
+    /// # Reverts
+    ///
+    /// * When there is a game playing.
```

Comment:
Can we be more specific here? i.e: does the function revert if there is any game currently playing, or only if there's already an active game between these same two players?
Also, consider this: given 2 players `A` and `B`, a game where `player_one = A` & `player_two = B` is fundamentally different than a game where `player_one = B` & `player_two = A`. Maybe it's better left to a future feature addition, but supporting both scenarios in parallel is a possibiity.

Reply:
There can only be 1 game played at a time. Once the game ends, via a draw or win, then the `new_game()` function can be called again.